lms/djangoapps/course_api/serializers.py

```diff
@@ -73,10 +75,24 @@ class CourseSerializer(serializers.Serializer):  # pylint: disable=abstract-meth
     pacing = serializers.CharField()
     mobile_available = serializers.BooleanField()
     hidden = serializers.SerializerMethodField()
+    course_modes = serializers.SerializerMethodField()
```

Comment:
so we get the price for the course. And we have to fill the seats for course_run in course_discovery.